File: app/src/main/java/jp/co/terraresta/androidlesson/data/model/profile_feed/ProfileFeedItem.java
package jp.co.terraresta.androidlesson.data.model.profile_feed;

import com.google.gson.annotations.Expose;

/**
 * Created by ooyama on 2017/05/26.
 */

public class ProfileFeedItem {

    @Expose
    private int userId;
    @Expose
    private String nickname;
    @Expose
    private int imageId;
    @Expose
    private String imageSize;
    @Expose
    private String imageUrl;

    public int getUserId() {
        return userId;
    }

    public String getNickname() {
        return nickname;
    }

    public int getImageId() {
        return imageId;
    }

    public String getImageSize() {
        return imageSize;
    }

    public String getImageUrl() {
        return imageUrl;
    }
}

File: app/src/main/java/jp/co/terraresta/androidlesson/data/model/profile_feed/ProfileFeedItem.kt
package jp.co.terraresta.androidlesson.data.model.profile_feed

import com.google.gson.annotations.Expose

/**
 * Created by ooyama on 2017/05/26.
 */

class ProfileFeedItem {

    @Expose
    val userId: Int = 0
    @Expose
    val nickname: String? = null
    @Expose
    val imageId: Int = 0
    @Expose
    val imageSize: String? = null
    @Expose
    val imageUrl: String? = null
}
